src/data/gui.rs

```diff
@@ -74,6 +75,9 @@ impl Gui {
                 }
             }
         } else {
+            if let Some(name) = block.get_field_value("name") {
+                self.widget_names.insert(name.to_string(), name.clone());
```

Comment:
Can replace with `name.as_str()` and use `&'static str`

Reply:
Will do